### PR TITLE
Add missing nixos rules

### PR DESCRIPTION
**Pull request by bjsowa**
Added some nixos rules to fix generating nix derivations for rolling packages.

The `flake8-builtins`, `flake8-comprehensions` and `flake8-quotes` python packages are not available for nixos but they prevent generating derivation for `ament_flake8` package which is a dependency of many essential packages (e.g. rclpy). That's why I decided to ignore them for now. `ament_flake8` works without these packages as they only add more checks to flake8 tool.

### CODE REVIEW

**Comment by wentasah** [comment]
LGTM. This increases the number of successfully generated packages in nix-ros-overlay for rolling from 1118 to 1138.

**Comment by nuclearsandwich** [comment]
> This increases the number of successfully generated packages in nix-ros-overlay for rolling from 1118 to 1138

Good for an [oblique reference](https://en.wikipedia.org/wiki/THX_1138) pre-[Star Wars Day](https://en.wikipedia.org/wiki/Star_Wars_Day)

**Review by nuclearsandwich — APPROVED** [review]

**Comment by nuclearsandwich** [comment]
Requires the second rosdep reviewer approval before merging.

**Review by clalancette — APPROVED** [review]

**Pull request merged**
